### PR TITLE
remove photon-capacitor-graviton from DSP recipes, closes #426

### LINKED ISSUE #426: Incorrect DSP recipes
The following recipes in DSP are incorrect. 

1. In version 0.6.16, we cannot produce critical photon with gravity lens any more. 
2. The two products of orbital collector should be separated to two different recipes, since they do not affect each other. 
    For example, when we are collecting on the ice giant, even if the collector is filled with fire ice, it can still collect hydrogen. 

I am trying to modify the `data.json` and will try to start a pull request later, but I am not quite familiar with them. So I think it is better to report it here in advance.

### PR DESCRIPTION
**Pull request by JiangengDong**
I can only fix the first problem in this issue #426. For the second problem, if I split the ice-giant recipe into 2, the solver gives me twice as many orbital collectors.

### CODE REVIEW

File: src/data/dsp/data.json [hunk 1]
```diff
@@ -1 +1 @@
...
...":1},"producers":["oil-extractor"]},{"id":"photon-capacitor-full","mining":true,"out":{"photon-capacitor-full":5},"time":60,"producers":["ray-receiver"]},{"id":"water-vein","mining":true,"time":1.2,"out":{"water":1},"producers":["water-pump"]},{"id":"sulphuric-acid-vein","mining":true,"time":1.2,"out":{"sulphuric-acid":1},"producers":["water-pump"]},{"id":"gas-giant","name":"Gasgiant","mining":true,"time":8,"out":{"hydrogen":4.08,"deuterium":0.17},"producers":["orbital-collector"]},{"id":"ice-giant","name":"Icegiant","mining":true,"time":8,"out":{"gas-hydrate":1.95,"hydrogen":0.84},"producers":["orbital-collector"]},{"id":"accumulator-full","name":"Accumulator(Full)","time":2,"in":{"accumulator":1},"producers":["energy-exchanger"]}],"defaults":{"modIds":[],"minBelt":"belt-1","maxBelt":"belt-3","fuel":"coal-ore","disabledRecipes":["ice-giant","nanotube-lv2","partical-capacitor-lv2","photo-shifter-lv2","silicium-high-purity-lv2","space-warper-lv2","casimir-crystal-lv2","diamond-lv2","gas-giant","graphene-lv2","crysrub-vein","fireice-vein","sulphuric-acid-vein"],"minFactoryRank":["assembler-1"],"maxFactoryRank":["assembler-3"],"moduleRank":[]},"limitations":{"productivity-module":[]}}
```
Comment (dcbroad3) [inline src/data/dsp/data.json:1]: It looks like your update removed all the spaces from strings. E.g. `position` fields went from `0px 0px` to `0px0px` and `name` from `Organic crystal` to `Organiccrystal`.

**Review by dcbroad3 — CHANGES_REQUESTED** [review]

**Comment by JiangengDong** [comment]
My apologize. 

Please close this thread and ignore my silly modification, because this is caused by my misunderstanding of the photon-capacitor-graviton recipe. What you mean is that using gravity lens can increase the efficiency of the ray receiver, but I took it as produce critical photon with gravity lens. 

I am so sorry for wasting you time.

**Comment by JiangengDong** [comment]
My apologize. 

Please close this thread and ignore my silly modification, because this is caused by my misunderstanding of the photon-capacitor-graviton recipe. What you mean is that using gravity lens can increase the efficiency of the ray receiver, but I took it as produce critical photon with gravity lens. 

I am so sorry for wasting you time.

**Comment by dcbroad3** [comment]
Aha! @JiangengDong thank you for following up. I wondered why they would remove it and had just confirmed that the recipe was in fact still in the game. 

No need to apologize! Thanks for taking the time to try to improve the calculator, even if it was just a misunderstanding!

**Pull request closed without merging**